x-pack/plugins/lens/public/xy_visualization/visualization.tsx

```diff
@@ -457,6 +458,7 @@ export const getXyVisualization = ({
             icon: undefined,
             lineStyle: 'solid',
             lineWidth: 1,
+            color: euiLightVars.euiColorDarkShade,
```

Comment:
small nit: Should we store the color as long as the user didn't change it or should we just keep it as undefined for this case (and handle it in to_expression and in the config UI)? This will allow us to change the default color later on. Same for line width and line style.

Reply:
++

Reply:
It was planned to refactor this part anyway, so I'll pick up the ball here